### PR TITLE
feat(ci): add `CODEOWNERS` & cleanup

### PR DESCRIPTION
**Pull request by kyrie25**
Cleanup unnecessary stuff, don't know why it was there.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @spicetify/marketplace
```
Comment (theRealPadster) [inline .github/CODEOWNERS:1]:
```suggest
* @spicetify/marketplace

```
Newline at EOF

File: packages/marketplace/src/components/ReadmePage.tsx [hunk 2]
```diff
@@ -107,7 +107,6 @@ class ReadmePage extends React.Component<
   }
 
   render() {
-    this.props.data.isInstalled();
```
Comment (theRealPadster) [inline packages/marketplace/src/components/ReadmePage.tsx:110]: Didn't you add this with the install button thing? I was assuming you needed the code to run for it to render or something, but I guess it's not necessary?

**Review by theRealPadster — CHANGES_REQUESTED** [review]

File: packages/marketplace/src/components/ReadmePage.tsx (hunk 2 above)
Comment (kyrie25) [reply packages/marketplace/src/components/ReadmePage.tsx:110]: It was when I was testing things, the function returns a boolean value that was assigned as state later. Having this actually breaks Marketplace occasionally.

File: .github/CODEOWNERS (hunk 1 above)
Comment (kyrie25) [reply .github/CODEOWNERS:1]: Will fix later.

**Review by theRealPadster — APPROVED** [review]

**Pull request merged**